blog/tests.py

```diff
@@ -118,3 +132,172 @@ def test_delete_blog(self):
         force_authenticate(request, user=user)
         response = view(request, slug=blog.slug)
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+    def test_update_blog(self):
+        """
+        Tests to update a Blog Post Successfully.
+        """
+        response = self.create_blog()
+        user = User.objects.get(username='test')
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        blog = Blog.objects.get()
+
+        body = {
+            'title': 'Test101',
+            'description': 'This is a Test Blog. Amirite! :D'
+        }
+
+        factory = APIRequestFactory()
+        url = '/api/blog/'
+
+        request = factory.put(url, body)
+        view = BlogAPI.as_view({
+            'put': 'update'
+        })
+
+        force_authenticate(request, user=user)
```

Comment:
Why we need to force_authenticate user every time? we already did once in the `_create_blog`

Reply:
because I make a separate request and I need to force_authenticate everytime a user is needed.